backend/grant/proposal/commands.py

```diff
@@ -90,36 +90,79 @@ def create_proposals(count):
 @with_appcontext
 def retire_v1_proposals(dry):
     now = datetime.datetime.now()
-    proposals = Proposal.query.filter_by(stage="FUNDING_REQUIRED").all()
-    modified_count = 0
+    proposals_funding_required = Proposal.query.filter_by(stage="FUNDING_REQUIRED").all()
+    proposals_draft = Proposal.query.filter_by(status=ProposalStatus.DRAFT).all()
+    proposals_pending = Proposal.query.filter_by(status=ProposalStatus.PENDING).all()
+    modified_funding_required_count = 0
+    modified_draft_count = 0
+    modified_pending_count = 0
 
-    if not proposals:
+    # TODO: what about APPROVED (before use publishes the proposal live) or STAKING stages?
```

Comment:
STAKING should get moved to pending

Reply:
APPROVED should get moved to pending, but we should make sure that it works well with v2 in that admins can select approve with or without funding.

Reply:
I'm thinking both `STAKING` and `APPROVED` would also have to go back to `DRAFT` status because the proposal target and milestones are going to have to be updated by the user.

Reply:
I think we'd have to notify them in this case, since they won't be expecting that they'll have to go update their proposals again.

Will the milestones be completely reset in this case then?

Reply:
As discussed, we can forgo modifying `STAKING` and `APPROVED` proposals as staking is being removed and there will be no proposals with an `APPROVED` status at the time of upgrade.